.rubocop_default_v1.yml

```diff
@@ -0,0 +1,299 @@
+########################################
+# EmploymentHero Shared Rubocop Config #
+########################################
+# rubocop ~> 1.15.0
+
+AllCops:
+  Exclude:
+    - 'db/schema.rb'
+    - 'node_modules/**/*'
+    - 'vendor/**/*'
+    - '.git/**/*'
+  EnabledByDefault: false
+  DisabledByDefault: false
+  NewCops: disable
```

Comment:
FYI, this is to disable NewCops. Rubocop has so many cops already and is on her way to get into departments